docs/source/process.rst

```diff
@@ -203,7 +183,7 @@ References
 
 .. [4] Demer DA, Berger L, Bernasconi M, Bethke E, Boswell K, Chu D, Domokos R,
    et al. (2015) Calibration of acoustic instruments. `ICES Cooperative Research Report No.
-   326. 133 pp. <https://doi.org/10.17895/ices.pub.5494>`_
+   1.   133 pp. <https://doi.org/10.17895/ices.pub.5494>`_
```

Comment:
Why is this changed to No. 1 ?  It is No. 326!

Reply:
Thanks for pointing this out. I was aware that my editor was doing this to this specific number for some reason (apparently every time I delete a line of text), but I missed it in the last commit.